server/entitymanager/entitymanager_test.go

```diff
@@ -11,11 +10,114 @@ import (
WFpbi1jZXJ0LXJldm9jYXRpb24tY2hlY2tzIjpmYWxzZX19oIIFozCCBZ8wggOHoAMCAQICAgfnMA0GCSqGSIb3DQEBCwUAMGExCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEOMAwGA1UEChMFQ2lzY28xHTAbBgNVBAMTFE1hbnVmYWN0dXJlciBSb290IENBMB4XDTIzMDgwOTIzNDkwNVoXDTMzMDgwOTIzNDkwNVowYTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MQ4wDAYDVQQKEwVDaXNjbzEdMBsGA1UEAxMUTWFudWZhY3R1cmVyIFJvb3QgQ0EwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQCT8XUaoChCJr7MxG5ay5dFJhfzo1OOMMUXij+cUaxaWBBXAsi6B24ka0sRvGeoPZoZiwy9/W5SaNTWC0CRTryDAwbuEnPgvM3aHCn0EydY6nS1O1cHVUHQ/wYEsfiDOc73MoYft0AvCmA1c+HF6YYZD5K4CAayePKfJ5uWSBHA5c7PjekDX2VjiFuI9Yq3R7EWePh3/ImTnF8rsagXoQ/KEWah6xm8LRS+NkqA/CBPON0QK2GGkjt21a7WRsgk5nph9Oaaai9E2kZyEe2+i3Tg4FujePKRgjmqpJBScvqguEWnceOnM/Smoj/M+f0YVt9qbNLLB6qMLBOtXFWpYNk9Vo5sWtDrDPn9m5w4KQrXmaxVMB7eJCAJmywvSKJprUA0vl6F2JPyjsJOEzOEiPHkDHOF37Ly/LDfNIf8hmIxbtaBNoTQXfH7BHKpp+Y2nd399lFFXlq4pBt/QFYSqM11YudFmQxlJvA3VwRl/ki8Ppq2XYOUwklDPHfUkqVPhRLkR7XOEfkkXMjBWqOvdbhn4igaFeUgCw0nwYmos+kmu6CpFL6uw+1hAuO6FBhdQmwVk87Ko9dqspRdMQ8wvmKiQVDfQjERmDTT1ObGdT+lZ0/B+OR8wHIu4yxLDE2hcg66I7e2gCkOGGOnJEJoTOMvtghY/a/kqOeEHn1nmEUrYQIDAQABo2EwXzAOBgNVHQ8BAf8EBAMCAoQwHQYDVR0lBBYwFAYIKwYBBQUHAwIGCCsGAQUFBwMBMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFDeNjw+UZS+oRo2uSBv+gBCtYEa4MA0GCSqGSIb3DQEBCwUAA4ICAQBebFIcMhgJtV7WJb9BLiHGR7Ju+KxALJJ9qigESLbobaxCuCS1mu6Lf6BOjzUIgViwwpTDYwXRoNKFCIUUif5pLs3gwWwtzA8mE7jB03lR+j/2muNf6DwdMQudB4VSvVOe5IfPtFsr9iDvn1dh98rAdEWoWEYmnFmx3gzLJlhDKNdIiCrQBW8gu/XGRXhi29bLrPYfv1+1fvlYCSkKegN3s48Aj4GKXbYRQzF7VRsr8X48iAz9gD5Bb5YHjpf0DM9YYXwzbltXT9Vk1HMYjCmnG9ysUT3IIdH52NBFDKxHQymMxyjxs8l2AgNYHE64BmqycAHDmDZGcXlwNR4IcEkxSIZc95Om3u/WUk3EgLDpC20fjCsWyq1f173ksSpjwgQ1f68jyUthSxtyYCi/zxVZG9HLEFQiwUvYLcmpHt7aLera3/wJfFLdmZOEVKQjX1IVpkvTSzNoO0+54QQUWOyxsvVv6EHlTnS7GvkwURG1oh+Jv3vWKUO4k1a6wYgCpJ/5ydxgP1O99cQcnB6IuoFgKKYXMtDNKj6sPUWT8uGuKRqb0avwBbZhNVGQUzaaTPrEke/DZIPs0CmdMqJmHNqUpv8lxRGh5iHsziyP+7HtFvz4XOuoFUW8JEIKvapPq5sJ5vEft6B1d40LYu+HrxHdkjY5tVkL3+wuS0pK+JZOxDGCAvkwggL1AgEBMGcwYTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MQ4wDAYDVQQKEwVDaXNjbzEdMBsGA1UEAxMUTWFudWZhY3R1cmVyIFJvb3QgQ0ECAgfnMAsGCWCGSAFlAwQCAaBpMBgGCSqGSIb3DQEJAzELBgkqhkiG9w0BBwEwHAYJKoZIhvcNAQkFMQ8XDTIzMDgwOTIzNDkwNlowLwYJKoZIhvcNAQkEMSIEIMF+hxaNZNSOUW96hqGA8Fufrgt5xrfo+rllZ2QwUmjyMAsGCSqGSIb3DQEBCwSCAgCQrVnM8lfVNnHx+/UHSO7iCYKMV9m9O7kSP3b66nvfuDvqTclgFrltUhmki0ECXcHVCYXUc4ts9w++T5Ga7sC3hLk/Pw4VD1hOr5IPmE3iZOzZD1JgrIxO9aqYVNW2S8ddUdNf4FQ7MwooCSRXEuJeesoTwmaBhgWpWnmmLz+7GSBZgvAnqZimqJnNxwYe0SycKslSispEjMp2wvHc98sjzUkPXzW5zn4PCNBb+gWTMNNH6Z29v+fZ5iItreeuueNSENoAda7uwdRA2v5PmEakrJzGh1zFOynMZeE5QaczDMVk6XREPBH1TYmNl+n/SMlo0J+7I1Tw2HliHn+x57o9mEWIVZWrapXcpVArAHR/R4NuJkWVY6ApKgN36fdYA5kNRv0ecnBn2H45xzQjjD0WGfHE8Zh61QiAgPQeUB58Sn0wRxZSQgwPXR74bTnYDBE4qDaSTEnOZD93gXxLEgjCxjkWncVjBHF0TGrMrWgYz0zyJSED1TISWy4hnL+eKTA7SOD40ZBfxxVZhU7GhHJYVahkFrhISlYEuWYcd/ZkzJacHKIOPUARL8f7F+CLkAMrq1PRULT0lG15gGA24DflFd3BhSEHQ+/zCt0UZEoVqqrAFbawLCKJyB5d6dIBqQoaVN0xnVwWIIlGnjOClHmDOvE2iQuyZgvTV3wKtPakVA=="
```

Comment:
Are these the same ovs that exist in the testdata directory? If so, can we just read them from the file instead of copying them here?

Reply:
done